drakrun/drakrun/util.py

```diff
@@ -133,6 +133,37 @@ def safe_delete(file_path) -> bool:
         return False
 
 
+def try_subprocess(list_args: list, msg: str, reraise=True, debug=True, **kwargs) -> subprocess.CompletedProcess:
```

Comment:
It would be a good idea to white some doc comment on how this function behaves i.e. that it throws on non-zero exit-code, captures output by default etc.

Reply:
Sure, will do